selector/global.go

```diff
@@ -2,12 +2,23 @@ package selector
 
 var globalSelector Builder
```

Comment:
var globalSelector  *wrapSelector

Reply:
It will conflict with https://github.com/go-kratos/kratos/blob/0f38511ea8303b42007b5bdf7b85ebd6fafc833f/transport/grpc/client.go#L27

Reply:
`var globalSelector = &wrapSelector{wrr.NewBuilder()}`

Can be initialized by default.

Reply:
In fact, we can't do it. import cycle not allowed

Reply:
wrr: init()